src/test/java/CovidLoveit/Service/Services/IndustryServiceTest.java

```diff
@@ -0,0 +1,228 @@
+package CovidLoveit.Service.Services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import CovidLoveit.Domain.InputModel.IndustryInputModel;
+import CovidLoveit.Domain.Models.Client;
+import CovidLoveit.Domain.Models.Industry;
+import CovidLoveit.Domain.Models.Role;
+import CovidLoveit.Exception.ClientException;
+import CovidLoveit.Exception.IndustryException;
+import CovidLoveit.Repositories.Interfaces.ClientRepository;
+import CovidLoveit.Repositories.Interfaces.IndustryRepository;
+import CovidLoveit.Repositories.Interfaces.RoleRepository;
+import CovidLoveit.Service.Services.ClientServiceImpl;
+import CovidLoveit.Service.Services.IndustryServiceImpl;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@ExtendWith(MockitoExtension.class)
+@DataJpaTest
+public class IndustryServiceTest {
+    @Autowired
+    private IndustryRepository industryRepository;
+
+    @Autowired
+    private ClientRepository clientRepository;
+
+    @Mock
+    private RoleRepository roleRepository;
+
+    @Mock
+    private AutoCloseable autoCloseable;
+    
+    @Mock
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    @InjectMocks
+    private IndustryServiceImpl industryService;
+
+    @InjectMocks
+    private ClientServiceImpl clientService;

```

Comment:
This is declared but never actually used in any of the test methods? Is it required in this case